backend/src/govsg/middlewares/govsg-template.middleware.ts

```diff
@@ -33,7 +33,7 @@ const convertMultilingualSupportToResponse = (
     language: languageSupport.language,
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
-    code: languageSupport.languageCode ?? languageSupport['language_code'],
+    code: languageSupport.language_code,
```

Comment:
Sequelize does not seem to care about nested fields, so they remain in snake case. Thus removing the redundant `xxx ??`